lib/conjur/command/ldapsync.rb

```diff
@@ -4,14 +4,16 @@ class Conjur::Command::LDAPSync < Conjur::Command
 
   LIST_FORMATS = %w(pretty json)
 
-  def self.find_job_by_id args
-    job_id = require_arg args, 'JOB-ID'
+  def self.error_messages(resp)
+    resp['events'].collect {|e| e['message'] if e['severity'] == 'error'}.compact
+  end
 
-    if (job = api.ldap_sync_jobs.find{|j| j.id == job_id})
-      job
-    else
-      exit_now! "No job found with ID '#{job_id}'"
-    end
+  def self.show_messages(resp)
+    msgs = resp['events'].collect do |e|
+      next unless e['severity'] == 'warn' || e['severity'] == 'error'
+      "\n" + e['severity'].upcase + ": " + e['message']
```

Comment:
I've run Rubocop at my last two companies, and I fear it's made me more hyper aware of code style. :)